fluentgen/src/main/resources/Readme.txt

```diff
@@ -45,15 +45,11 @@ Azure Management Libraries require a `TokenCredential` implementation for authen
 
 ### Authentication
 
-By default, Microsoft Entra ID token authentication depends on correct configuration of the following environment variables.
+Microsoft Entra ID token authentication relies on the [credential class](azure_identity_credentials) from [Azure Identity][azure_identity] package.
```

Comment:
It's safer to direct user to azure-identity doc..

Reply:
`[credential class][azure_identity_credentials]` instead of `[credential class](azure_identity_credentials)`?